app/javascript/react/components/Map/Map.tsx

```diff
@@ -217,6 +217,25 @@ const Map = () => {
             unitSymbol={unit_symbol}
           />
         )}
+        {!fixedSessionTypeSelected ? (
+          <MobileMarkers
+            sessions={sessionsPoints}
+            onMarkerClick={handleMarkerClick}
+            selectedStreamId={selectedStreamId}
+          />
+        ) : (
+          <MobileMarkers
+            sessions={sessionsPoints}
+            onMarkerClick={handleMarkerClick}
+            selectedStreamId={selectedStreamId}
+          />
+        )}
+        {selectedStreamId && !fixedSessionTypeSelected && (
+          <StreamMarkers
+            sessions={mobileStreamPoints}
+            unitSymbol={unit_symbol}
+          />
+        )}
```

Comment:
I think this shouldn't be in this PR. I'll clean up this in `mobile session fixes` 💪